### PR TITLE
Fix #64 - Replace ImportMultiSigContractAddr

### LINKED ISSUE #64: claim script incompatible with new neo-python release
The claim_neo_and_gas script doesn't work anymore with the new neo-python release. Here's the claim script: https://github.com/CityOfZion/neo-privatenet-docker/blob/master/scripts/claim_neo_and_gas_fixedwallet.py

The error is 

    Traceback (most recent call last):
      File "/neo-python/claim_neo_and_gas_fixedwallet.py", line 43, in <module>
        from neo.Prompt.Commands.LoadSmartContract import ImportMultiSigContractAddr
    ImportError: cannot import name 'ImportMultiSigContractAddr'

Currently I've updated the [Dockerfile](https://github.com/CityOfZion/neo-privatenet-docker/blob/master/Dockerfile#L41) to use neo-python v0.8.2. If you want to work at this, comment this line to use the master branch of neo-python.

### PR DESCRIPTION
**Pull request by LysanderGG**
Fix for #64

Tested this time 😅 
I replaced the call to ImportMultiSigContractAddr by the main lines of its implementation.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -38,7 +38,6 @@ RUN rm -rf /var/lib/apt/lists/*
 RUN git clone https://github.com/CityOfZion/neo-python.git /neo-python
 WORKDIR /neo-python
 # RUN git checkout development
-RUN git checkout v0.8.2
```
Comment (jseagrave21) [inline Dockerfile:41]: Is the intention still to checkout development as the comment implies? If so I recommend replacing with `RUN git checkout development`

Comment (jseagrave21) [reply Dockerfile:41]: I think I see from https://github.com/CityOfZion/neo-privatenet-docker/issues/64#issue-400713932 that the master branch is desired. If so, why leave the commented code in the released product?

Comment (LysanderGG) [reply Dockerfile:41]: This is basically reverting https://github.com/CityOfZion/neo-privatenet-docker/commit/708f6d3a400f112da24f80e4aa5b6c7fc3b63b9e

**Comment by metachris** [comment]
Thanks! 👍

**Pull request merged**